Documentation/operations/troubleshooting.rst

```diff
@@ -885,12 +686,12 @@ When running in :ref:`arch_overlay` mode:
      other.
    * verify that the firewall on each node allows UDP port 8472.
 
-When running in :ref:`arch_direct_routing` mode:
+   When running in :ref:`arch_direct_routing` mode:
 
-3. Run ``ip route`` or check your cloud provider router and verify that you have
+5. Run ``ip route`` or check your cloud provider router and verify that you have
```

Comment:
Incidentally, with the fixed spacing above, these numbers should be numbered correctly with `#.`.